subsys/net/lib/zperf/zperf_udp_receiver.c

```diff
@@ -347,6 +351,10 @@ static int zperf_udp_receiver_init(void)
 			in4_addr_my->sin_addr.s_addr = INADDR_ANY;
 		}
 
+		if (net_ipv4_is_addr_mcast(&in4_addr_my->sin_addr))
+			zperf_udp_join_mcast_ipv4(udp_server_iface_name,
+				&in4_addr_my->sin_addr);
+
```

Comment:
This seems to be belong to some other PR, please remove it. Other than that, LGTM.

Reply:
yes, sry removed